js/scripts.js

```diff
@@ -0,0 +1,17 @@
+// Excercise 8.6
+// isNaN method added 
+
+var a = prompt("podaj liczbę");  
+var b = prompt("podaj liczbę"); 
+var number = (a * a) + (2 * a * b) - (b * b); 
+console.log(number);
+
+if (number > 0) {
+	alert(number + 'liczba dodatnia');
+} else if (number < 0) {
+	alert(number + 'liczba ujemna');
+} else if (number == isNaN(number)) {
```

Comment:
every if conditional returns you true or false - when false it will go to another `if`, when true it will come in. so isNaN() method also return you true or false (this is information about number you are checking is Not a Number).

to this part you will enter only when number = 0, because `0 == false` :)